lib/SILOptimizer/Utils/CanonicalizeOSSALifetime.cpp

```diff
@@ -159,7 +167,7 @@ bool CanonicalizeOSSALifetime::computeCanonicalLiveness() {
           continue;
         }
       }
-      switch (use->getOperandOwnership()) {
+      switch (auto ownership = use->getOperandOwnership()) {
```

Comment:
Cool, I didn't know this pattern worked. Where are you using the value?

Reply:
I was going to use it in the debug message below, but I forgot to. I can remove it.